onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -32,15 +31,53 @@ def _skip_normalization(op, input, skip, gamma, epsilon, stash_type):
     return normalized, skip_sum
 
 
-_rule = pattern.RewriteRule(
-    _skip_norm_pattern, _skip_normalization, matcher=pattern.SimplePatternMatcher
+_skip_rms_rule = pattern.RewriteRule(
+    _skip_rms_norm_pattern, _skip_rms_normalization, matcher=pattern.SimplePatternMatcher
```

Comment:
Curious: do we need to specify the pattern-matcher here?

Reply:
Not needed, removed.